### ISSUE TITLE
Serviio 2.2

### ISSUE THREAD

**Issue by mr-simba** [rb-dock8s/serviio#20]
Will there be a release for the new Serviio 2.2?

https://www.serviio.org/news/122-version-2-2-released

**Comment by ergoz** [comment]
Yes, shure. I think in few hours :)

пт, 1 окт. 2021 г., 15:43 mr-simba ***@***.***>:

**Comment by mr-simba** [comment]
Thanks for updating. But on docker hub there is still only 2.1 available:
https://hub.docker.com/r/riftbit/serviio

**Comment by ergoz** [comment]
Yes, i know. I will fix building. Problem on docker hub side, they are
closed free builds, i uses free account there :) i will move this sources
to gitlab and will use gitlab ci. This repo will be used for issues and
feedback :)

вс, 3 окт. 2021 г., 17:09 mr-simba ***@***.***>:

**Comment by ralphklos** [comment]
Thank you for your effort in this, i was able to build the docker image fine from the dockerfile.

**Comment by geezer500** [comment]
Is the update to 2.2 ready? if so how do I update? thx
(I've tried docker pull riftbit/docker-serviio:latest... I keep getting 
"_Error response from daemon: pull access denied for riftbit/docker-serviio, repository does not exist or may require 'docker login': denied: requested access to the resource is denied_")

**Comment by ralphklos** [comment]
you can download the tar.gz file, untar  and from that folder run docker build -t serviio2 .
This will create the docker image, which you can run with the same parameters as you did do with the old one.

**Comment by geezer500** [comment]
@ralphklos Thanks for that... Worked perfectly! (I wasn't really sure how to update as I've never built an image before)

**Comment by HaJointheNet** [comment]
Sorry, I'm not able to run the build process, is there a chance to find a ready to use image

**Comment by soerentsch** [comment]
Hi, i had the same issue with the missing update. For my personal use i created an updated fork with Serviio 2.2.1 and FFmpeg 5.0.
Feel free to use it.
You can find the Docker file [here](https://github.com/soerentsch/docker-serviio)
And the Docker image at Docker Hub [here](https://hub.docker.com/u/soerentsch/serviio)

**Comment by HaJointheNet** [comment]
> Hi, i had the same issue with the missing update. For my personal use i created an updated fork with Serviio 2.2.1 and FFmpeg 5.0.
> Feel free to use it.

Thank you! Working really fine for me!
Is there a possibility to do a update to your newest version without reinstalling/configure it complete new on Synology?

**Comment by soerentsch** [comment]
> > Hi, i had the same issue with the missing update. For my personal use i created an updated fork with Serviio 2.2.1 and FFmpeg 5.0.
> > Feel free to use it.
> 
> Thank you! Working really fine for me! Is there a possibility to do a update to your newest version without reinstalling/configure it complete new on Synology?

Yes, if you have map the /opt/serviio/library (optional also the plugins and log path) and your media paths outsite the container.
